src/graph.ts

```diff
@@ -17,7 +17,7 @@ export interface INode extends IShareableNode {
   extensions?: Extensions;
 }
 
-export interface INodeTag {
```

Comment:
couldn't this also be on a `IComponentNode`?

Reply:
what do you mean?

Reply:
if you're ok with it, can I get a 👍 ?